main.js

```diff
@@ -1,59 +1,64 @@
-/* eslint-disable  */
-const addBookForm = document.getElementById('add-book-form');
-const booksList = document.getElementById('books-list');
-
-let books = JSON.parse(localStorage.getItem('books')) || [];
-
-// display the books list 
-books.forEach((book) => {
-  const bookElement = createBookElement(book.title, book.author);
-  booksList.appendChild(bookElement);
-});
-
-addBookForm.addEventListener('submit', (event) => {
-  event.preventDefault();
-
-  // Get the input values
-  const titleInput = document.getElementById('title');
-  const authorInput = document.getElementById('author');
-  const title = titleInput.value;
-  const author = authorInput.value;
-
-  // Add the new book
-  const newBook = { title, author };
-  books.push(newBook);
-
-  // Save the updated books array
-  localStorage.setItem('books', JSON.stringify(books));
-
-  const bookElement = createBookElement(title, author);
-  booksList.appendChild(bookElement);
-
-  // Reset the form
-  addBookForm.reset();
-});
+/* eslint-disable */ 
```

Comment:
- [ ] Kindly consider avoiding disabling the linter because it helps keep code tidy and easier to understand and use throughout the project for anyone who needs to look at the project code in the future